service/awsconfig/v5/resource/cloudformation/adapter/outputs_test.go

```diff
@@ -3,41 +3,159 @@ package adapter
 import (
 	"testing"
 
-	// NOTE(PK): This import is disturbing. I'm not bothering. It's first candidate to go away.
-	"github.com/giantswarm/aws-operator/service/awsconfig/v5/cloudconfig"
```

Comment:
We should be explicit in tests. When you change something you might have no idea what you are doing and tests should tell you what is going on. Right now you change the something and the tests are just fine because they use the reference you just changed.